.changeset/few-schools-run.md

```diff
@@ -0,0 +1,5 @@
+---
+"@livekit/agents": patch
+---
+
+add tests + fix tokenizer
```

Comment:
should the tests bit be part of the changelog?

Reply:
nope, fixing